packages/blade/src/components/Input/BaseInput/StyledBaseInput.web.tsx

```diff
@@ -34,9 +34,8 @@ const getWebInputStyles = (
       ...getTextStyles({
         size: 'medium',
         variant: 'body',
-        type: 'placeholder',
         weight: 'regular',
-        contrast: 'low',
```

Comment:
TODO: Document any prop removals/replacements in the migration doc for codemod development.